### ISSUE TITLE
AVT 3 - Loading and inline loading components are not announced by VoiceOver

### ISSUE THREAD

**Issue by snidersd** [IBM/carbon-components-react#2191]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
VO does not announce the loading state.

This issue is similar to the [Carbon Component issue#1906](https://github.com/IBM/carbon-components/issues/1906)
